codalab/lib/bundle_cli.py

```diff
@@ -2012,6 +2013,44 @@ def do_rm_command(self, args):
             for uuid in deleted_uuids:
                 print(uuid, file=self.stdout)
 
+    @Commands.command(
+        'ancestors',
+        help='Find ancestors of a bundle.',
+        arguments=(
+            Commands.Argument(
+                'bundle_spec', help=BUNDLE_SPEC_FORMAT, nargs='+', completer=BundlesCompleter
+            ),
+            Commands.Argument(
+                '-w',
+                '--worksheet-spec',
+                help='Operate on this worksheet (%s).' % WORKSHEET_SPEC_FORMAT,
+                completer=WorksheetsCompleter,
+            ),
```

Comment:
What is `indent_count`? Maybe you should add it to `arguments` so that it's documented.

Reply:
It's supposed to be internal - it lets the function know how many indents to make based on how many parents there are, just like in the example. If bundle z depends on x and y, and x depends on foo, it would look like:

- z
  - x
    - foo
  - y

Foo is 2 layers in, so there would be two indents.

Do we want the user to be able to set how many indents out the output starts?

Reply:
Actually, should probably prefix with xxx to convey it's internal

Reply:
Ah, I see, thanks!